cmd/soroban-cli/src/events.rs

```diff
@@ -196,8 +195,8 @@ impl Cmd {
         }
 
         let events = match (self.rpc_url.as_ref(), self.events_file.as_ref()) {
-            (Some(rpc_url), _) => self.run_against_rpc_server(rpc_url).await,
-            (_, Some(path)) => self.run_in_sandbox(path),
+            (Some(rpc_url), _) => self.run_against_rpc_server(rpc_url, start).await,
+            (_, Some(path)) => self.run_in_sandbox(path, start),
```

Comment:
[required_unless_present](https://docs.rs/clap/latest/clap/struct.Arg.html#method.required_unless_present) could also be used here (but it's unrelated to this PR)